chain/indexer/Cargo.toml

```diff
@@ -13,7 +13,7 @@ anyhow = "1.0.51"
 async-recursion = "0.3.2"
 tracing = "0.1.13"
 futures = "0.3.5"
-rocksdb = { version = "0.18.0", default-features = false, features = ["snappy", "lz4", "zstd", "zlib"] }
+rocksdb = { git = "https://github.com/near/rust-rocksdb", rev = "18355460e0dbecbb79025bec4637c4d707b962ed", default-features = false, features = ["snappy", "lz4", "zstd", "zlib"] }
```

Comment:
Should we create a PR on the rust-rocksdb repo and get some eyes on it as well?

Reply:
We can't create a PR in rust-rocksdb until the changes in rocksdb are in.

Reply:
I meant a PR for someone at NEAR to review.  So you could create the PR in a branch on your fork against the master branch on your fork and then ask a couple of people in the team to review it.

Reply:
You can see the diff [here](https://github.com/rust-rocksdb/rust-rocksdb/compare/master...near:rust-rocksdb:use_near_rocksb_expose_lru_num_shards_and_checksum_type). Note the submodule change as well.